smithy-swift-codegen/build.gradle.kts

```diff
@@ -34,6 +38,8 @@ dependencies {
     implementation("software.amazon.smithy:smithy-aws-traits:$smithyVersion")
     testImplementation("org.junit.jupiter:junit-jupiter:$junitVersion")
     testImplementation("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
+    implementation("software.amazon.smithy:smithy-rules-engine:$smithyVersion")
+    implementation("software.amazon.smithy:reterminus-core:0.2.0")
```

Comment:
**Correctness**: I cannot find **reterminus-core** [listed in Maven Central](https://repo1.maven.org/maven2/software/amazon/smithy/). Is this replaced by **smithy-rules-engine**?

Reply:
Correct, it will be part of the smithy-rules-engine.

There is already a PR https://github.com/awslabs/smithy/pull/1356 for it.